test/e2e/utils.go

```diff
@@ -221,3 +266,17 @@ func containerFound(containers []*runtimeapi.Container, containerID string) bool
 	}
 	return false
 }
+
+func checkLog(podConfig *runtimeapi.PodSandboxConfig, logPath string, expectedContext string) {
```

Comment:
@feiskyer You mean I should import `github.com/docker/docker/daemon/logger`? Dose this pkg have the function which can verify log contents?